src/wp-includes/cache-compat.php

```diff
@@ -35,3 +35,86 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 		return $values;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_delete_multiple' ) ) :
+	/**
+	 * Retrieves multiple values from the cache in one call.
+	 *
+	 * Compat function to mimic wp_cache_delete_multiple().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_delete_multiple()
+	 *
+	 * @param array  $keys  Array of keys under which the cache to deleted.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @return array Array of return values organized into groups.
+	 */
+	function wp_cache_delete_multiple( $keys, $group = '' ) {
```

Comment:
What about atomicity?

Should we be adding an optional parameter to indicate that we'd want the call to be atomic? Whether the cache implementation supports or is able to, it is up to it.

Reply:
We can leave this for a follow-up PR.